src/sentry/models/integration.py

```diff
@@ -77,6 +77,7 @@ class OrganizationIntegration(DefaultFieldsModel):
     status = BoundedPositiveIntegerField(
         default=ObjectStatus.VISIBLE, choices=ObjectStatus.as_choices()
     )
+    date_to_disable = models.DateTimeField(null=True, blank=True)
```

Comment:
@scefali what do you think of `grace_period_end`? I don't think `grace_period_start` makes sense. I prefer having the last date active to query against so that the notify task just needs to check if current_date > grace_period_end. Also it'll be easier to add functionality for Support to extend the grace period if necessary

Reply:
@NisanthanNanthakumar We can always do `grace_period_start + grace_period_length > current_date` in the query which is simple enough. But good point about extending the grace period by support...didn't think about that.